st2common/st2common/content/bootstrap.py

```diff
@@ -219,8 +257,23 @@ def register_content():
     if cfg.CONF.register.policies:
         register_policies()
 
+    if cfg.CONF.register.setup_virtualenvs:
+        setup_virtualenvs()
+
 
 def setup(argv):
+    if '--register-setup-virtualenvs' in argv:
+        # If setup-virtualenvs option is used we also want to setup config for action runner since
+        # it contains some setup virtualenv related options.
+        # Keep in mind that using "setup-virtualenv" only makes sense on nodes which run action
+        # runners.
+        # Note: This is a hack, but we can't check if that flag is provided using oslo config since
+        # we need to parse config first.
+        from st2actions import config as action_runner_config
```

Comment:
Can you put this in a try ... except to print a useful exception message? It would be nice to know that module import failed because st2actions is not present.

Another alternative is to move the action_runner_config to st2common. We have already done that in a few cases. Again not ideal but possible.